### DISCUSSION TITLE
QBEE新版本已经支持网页的shadowban api了

### DISCUSSION THREAD

**Discussion by Clackz** [PBH-BTN/PeerBanHelper#377] [讨论]
QBEE新版本已经支持网页的shadowban api了，作者有打算支持这个功能吗？

**Comment by zhongfly** [comment]
https://github.com/c0re100/qBittorrent-Enhanced-Edition/pull/568#issuecomment-2236440286

开发者在对应的pr留过言，已经注意到这个了…估计会考虑的
